docs/manual/docs/cmd/actions/login.md

```diff
@@ -0,0 +1,20 @@
+# login action
+
+This action helps to login into a tenant.
```

Comment:
Could you please add `GitHub ` so it is `GitHub action`?

Reply:
Will do when I update the PR.